components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -126,6 +126,7 @@ export const BlurryPoster = ({
           aria-describedby={hasCaption ? 'story-hero-caption' : undefined}
           className={styles.bgImage}
           fetchPriority="high"
+          loading="lazy"
```

Comment:
Blurry Poster is also used for Story Heroes, and I didn't like how the lazy loading works even when not used in the Story Hero since they're on the homepage and have some animation with the images 😄 So let's remove this one.